kod/object/active/holder/room/battlearena.kod

```diff
@@ -0,0 +1,1095 @@
+   room_BattleArena = MarDun02.roo
+   BattleArena_music = pk_fm.mid
+
+   BattleArena_door_sound = stoneup2.wav
+
+   no_participation_spectators = "As a spectator, you may not attack or "
+                                 "cast in any manner."
+   no_attack_teammate = "You can't attack a member of your own team!"
+   cannot_cast_that_here = "You can't cast that spell here."
+   
+   welcome_blurb = "### Welcome to the Battle Arena!"
+   info_blurb = "### You are a spectator. Say 'join' to enter the fray, "
+                "or say 'leave the battle arena' to return to your last safe "
+                "location. State 'jackpot' to see the current prize pool, "
+                "'flags' for control information, and 'time' for the time "
+                "remaining until victory."
+   
+   info_blurb_two = "### This arena contains nine flags. An individual "
+                    "team member can only carry up to three. Therefore, "
+                    "your team must work together to carry all nine flags "
+                    "for three minutes to win the jackpot. Good luck!"
+
+   assigned_team_zandramas = "### You have been assigned to Team Zandramas! "
+                             "Say 'spectate' to return to spectating."
+   assigned_team_zaphod = "### You have been assigned to Team Zaphod! "
+                          "Say 'spectate' to return to spectating."
+   
+   report_team_spectator = "You are a spectator."
+   report_team_zandramas = "You are on Team Zandramas."
+   report_team_zaphod = "You are on Team Zaphod."
+   report_team_none = "You are not on any team, and are probably a cheater."
+   
+   kill_reward_jackpot = "### %s has slain %s, increasing the jackpot to %i!"
+   kill_reward_individual = "### For killing %s, you have been awarded "
+                            "%i shillings."
+   current_jackpot = "### The jackpot currently stands at %i shillings."
+   flags_info = "### Team Zandramas controls %i flags, Team Zaphod controls "
+                "%i flags, and %i flags remain neutral."
+   flag_distributed = "### A flag has been randomly distributed."
+   time_info_no_control = "### Currently, neither team controls all the flags."
+   time_info_zandramas = "### %i seconds remain until Team Zandramas wins!"
+   time_info_zaphod = "### %i seconds remain until Team Zaphod wins!"
+   victory_aborted = "### Flag victory aborted due to loss of one or "
+                     "more flags."
+   victory_achieved_zandramas = "### Team Zandramas has won, receiving %i "
+                                "shillings each!"
+   victory_achieved_zaphod = "### Team Zaphod has won, receiving %i "
+                             "shillings each!"
+   loss_reward = "### You've been awarded %i consolation shillings for "
+                 "your team's Battle Arena loss."
+   
+   BattleArena_no_offer = "You can't trade items here!"
+
+   JOIN_COMMAND = "join"
+   SPECTATE_COMMAND = "spectate"
```

Comment:
These are resources - should be lower case so as not to confuse them with constants.